src/image-viewer.js

```diff
@@ -325,18 +325,18 @@ export class ImageViewer {
 
   /** @private */
   setupGestures_() {
-    const gesturesWithoutPreventDefault = Gestures.get(this.image_,
+    this.gestures_ = Gestures.get(this.viewer_,
         /* opt_shouldNotPreventDefault */true);
-    gesturesWithoutPreventDefault.onPointerDown(() => {
+    this.gestures_.onPointerDown(() => {
       if (this.motion_) {
         this.motion_.halt();
+        event.preventDefault();
       }
     });
 
-    this.gestures_ = Gestures.get(this.viewer_);
-
     // Zoomable.
     this.gestures_.onGesture(DoubletapRecognizer, e => {
+      event.preventDefault();
```

Comment:
I think you want `e.preventDefault`

Reply:
No actually, the e that we get from `GestureRecognizers` isn't an event. It's an object that contains custom read-only data pertaining to each gesture. For example, the one emitted by the `DoubletapRecognizer` is: 

`    this.signalEmit({clientX: this.lastX_, clientY: this.lastY_}, null);`